contracts/Vault.sol

```diff
@@ -137,6 +137,13 @@ contract Vault is
     //
     // IVault
     //
+    function setTreasury(address _treasury)
+        external
+        override(IVault)
+        requiresTrust
```

Comment:
you'll need to change this trust part though. it's been refactored in favor of AccessControl